substrate/client/rpc-api/src/author/error.rs

```diff
@@ -86,98 +83,104 @@ const POOL_NO_TAGS: i32 = POOL_INVALID_TX + 9;
 const POOL_INVALID_BLOCK_ID: i32 = POOL_INVALID_TX + 10;
 /// The pool is not accepting future transactions.
 const POOL_FUTURE_TX: i32 = POOL_INVALID_TX + 11;
+/// Other error.
+const OTHER_ERR: i32 = BASE_ERROR + 40;
```

Comment:
How do we ensure not to use the same error code twice? Eg `BASE_ERROR + 2` comes up in a separate file.

Reply:
Aah, It's a different BASE_ERROR!

Reply:
Your comment makes sense anyway, globally there is nothing that checks that each error gets unique error code.

It's a manual thingy ^